Analyse_Birds.qmd

```diff
@@ -107,13 +117,220 @@ required_packages <- c(
 # Install and load the packages
 install_if_missing(required_packages)
 ```
 
+# First look at the data : 
+
+You can visualise the first rows of the dataset by running the following cell :
+```{r}
+# View the firsts lignes of the dataset
+head(data)
+```
+
+In order to picture the data in a more visual way, we will be representing our dataset on a map. 
+The first step in our map creation is the separation of the dataset by the counties. 
+
+```{r}
+data_list_county <- split(data, data$County)
+
+data_will <- data_list_county[["Will"]]
+data_jo_daviess <- data_list_county[["Jo Daviess"]]
+data_lafayette <- data_list_county[["Lafayette"]]
+data_iowa <- data_list_county[["Iowa"]]
+data_dane <- data_list_county[["Dane"]]
+data_green <- data_list_county[["Green"]]
+data_grant <- data_list_county[["Grant"]]
+data_grundy <- data_list_county[["Grundy"]]
+data_lee_ogle <- data_list_county[["Lee/Ogle"]]
+data_lee <- data_list_county[["Lee"]]
+data_rock <- data_list_county[["Rock"]]
+data_monroe <- data_list_county[["Monroe"]]
+data_carroll <- data_list_county[["Carroll"]]
+data_carroll_whiteside <- data_list_county[["Carroll/Whiteside"]]
+```
+
+We are now adding columns for the coordinates of each county, the coordinates were obtained by searching for the conties on openstreetmap. 
+
+```{r}
+data_will <- data_will %>%
+  mutate(longitude_column = -88.0817,
+         latitude_column = 41.5250)
+
+data_jo_daviess <- data_jo_daviess %>%
+  mutate(longitude_column = -90.4290,
+         latitude_column = 42.4167)
+
+data_grundy <- data_grundy %>%
+  mutate(longitude_column = -88.4210,
+         latitude_column = 41.3570)
+
+data_lee <- data_lee %>%
+  mutate(longitude_column = -89.4850,
+         latitude_column = 41.8430)
+
+data_carroll <- data_carroll %>%
+  mutate(longitude_column = -89.9826,
+         latitude_column = 42.0956)
+
+data_lafayette <- data_lafayette %>%
+  mutate(longitude_column = -90.1160,
+         latitude_column = 42.6825)
+
+data_iowa <- data_iowa %>%
+  mutate(longitude_column = -90.1160,
+         latitude_column = 42.6825)
+
+data_dane <- data_dane %>%
+  mutate(longitude_column = -89.4012,
+         latitude_column = 43.0731)
+
+data_green <- data_green %>%
+  mutate(longitude_column = -89.6387,
+         latitude_column = 42.6017)
+
+data_grant <- data_grant %>%
+  mutate(longitude_column = -90.7082,
+         latitude_column = 42.8475)
+
+data_rock <- data_rock %>%
+  mutate(longitude_column = -89.0187,
+         latitude_column = 42.6828)
+
+data_monroe <- data_monroe %>%
+  mutate(longitude_column = -90.8129,
+         latitude_column = 43.9447)
+
+data_lee_ogle <- data_lee_ogle %>%
+  mutate(longitude_column = -89.4038,
+         latitude_column = 41.9293)
+
+data_carroll_whiteside <- data_carroll_whiteside %>%
+  mutate(longitude_column = -89.9742,
+         latitude_column = 41.9527)
+```
+
+Now, for each county dataset, we subdivide the said dataset for each year when at least one nest has been observed in this conty. 
```

Comment:
Same : "county"

Reply:
This typo has also been updated, thank you for spotting it